docs/api/autogluon.task.rst

```diff
@@ -9,22 +9,20 @@ AutoGluon Tasks
    Tell AutoGluon that task is image classification:
 
    >>> import autogluon.core as ag
-   >>> from autogluon.vision import ImageClassification as task
+   >>> from autogluon.vision import ImageClassification as Task
 
    Load a toy image dataset:
 
-   >>> filename = ag.download('http://autogluon-hackathon.s3.amazonaws.com/data.zip')
-   >>> ag.unzip(filename)
-   >>> dataset = task.Dataset(train_path='data/train')
+   >>> train, val, test = Task.Dataset.from_folders('http://autogluon-hackathon.s3.amazonaws.com/data.zip')
```

Comment:
nit: probably want to move datasets in the hackathon S3 bucket to the main `autogluon` S3 bucket to avoid inconsistencies.

Reply:
Fixed